addons/web_dashboard/static/src/js/dashboard.js

```diff
@@ -0,0 +1,256 @@
+odoo.define('dashboard.dashboard', function (require) {
+"use strict";
+
+var core = require('web.core');
+var Widget = require('web.Widget');
+var ajax = require('web.ajax');
+var Planner = require('planner.planner');
+
+var QWeb = core.qweb;
+var _t = core._t;
+
+var Dashboard = Widget.extend({
+    template: 'DashboardMain',
+    events: {
+        'click .install_new_apps': 'new_apps',
+    },
+    start:function(){
+        var self = this;
+        ajax.jsonRpc('/web/dataset/call', 'call', {
+            model: 'ir.model.data',
+            method: 'xmlid_to_res_id',
+            args: ['base.view_users_form'],
+        }).then(function(res_id) {
+            self.user_form_id = res_id;
+            self.load();
+        });
+    },
+    load:function(){
+        var self = this;
+        ajax.jsonRpc("/dashboard/info", 'call', {}).then(function (data) {
+            self.$el.find('#apps').append(QWeb.render("DashboardApps", data));
+            new DashboardInvitations(self,data).appendTo('#invitations');
+            new DashboardPlanner(self,data).appendTo('#planner');
+            new DashboardVideos(self,data).appendTo('#videos');
+        });
+    },
+    new_apps:function(){
+        this.do_action('base.open_module_tree', {
+            'additional_context': {'search_default_app':1,'search_default_not_installed':1}
+        });
+    },
+    do_reload:function(){
+        this.$el.find('.col > div').children().remove();
+        this.load();
+        $('.loader').addClass("hidden");
```

Comment:
There is already a loader in Odoo. Why adding a second one ? What is its specialty ?